codegen/templates/manifests/crd.yamltmpl

```diff
@@ -1,6 +1,6 @@
 [[- range $crd := .Crds -]]
 [[- if render_outer_conditional_crd_template $crd (index $crd.Spec.Versions 0).Name $.ShouldSkip ]]
-{{- if has "[[ $crd.GetName ]]/[[ (index $crd.Spec.Versions 0).Name ]]" $.Values.[[ $.EnabledAlphaApiFlagName ]] }}
+{{- if or (not has "[[ $crd.Spec.Group ]]/[[ $crd.Spec.Names.Kind ]]" $.Values.[[ $.DisabledApiFlagName ]]) (has "[[ $crd.GetName ]]/[[ (index $crd.Spec.Versions 0).Name ]]" $.Values.[[ $.EnabledAlphaApiFlagName ]]) }}
```

Comment:
There's also a template code here: https://github.com/solo-io/skv2/blob/e28e804e83b0a5c6ebade344c92ffaea68ce3f48/codegen/templates/manifests/crd.yamltmpl#L18

That would be used in multi-versioned CRDs. We don't have them atm though, not sure if we'll ever have them. Perhaps instead of updating that piece of the template we should just remove it?